tests/func/test_remote.py

```diff
@@ -257,3 +258,33 @@ def test_modify_missing_remote(dvc):
 
     with pytest.raises(ConfigError, match=r"unable to find remote section"):
         remote_config.modify("myremote", "gdrive_client_id", "xxx")
+
+
+def test_verify_checksums(tmp_dir, erepo_dir, mocker):
+    with erepo_dir.chdir():
+        erepo_dir.dvc_gen({"file": "file1 content"}, commit="add file")
+        erepo_dir.dvc_gen(
+            {"dir": {"subfile": "file2 content"}}, commit="add " "dir"
+        )
+
+    Git.clone(fspath(erepo_dir), fspath(tmp_dir))
+
+    from dvc.repo import Repo
+
+    dvc = Repo(fspath(tmp_dir))
+
```

Comment:
Not sure I understand why this is needed. COuld you elaborate, please?

Reply:
Well, to perform `pull` operation on "clear" repo I can either:
- create repo,
- create remote,
- create elements, add them and push
- remove local cache, remove local files
- perform pull
or:
- create elements in erepo
- clone it
- pull
 Second option seems faster

Reply:
>     Git.clone(fspath(erepo_dir), fspath(tmp_dir))

will probably clone `erepo_dir` into `tmp_dir / "some_dir"`, or am i missing something?

Reply:
Ah, yeah, it will clone to `tmp_dir` directly. Ok, need to give it a fresh look, so far I'm confused.

Reply:
Not quite, we wouldn't be able to initialize `Repo` with actual repo being in `some_dir` that would be subdir of `tmp_dir`. Also, double-checked `.dvc` is inside `tmp_dir`.

Reply:
@pared Let's do it the other way around, please. I have to give it a very deep thought to figure out what is going on in this test, because it implies that erepo has remote already setup (which we will remove soon, as we have an auto-remote logic already).